modules/javafx.graphics/src/main/java/com/sun/glass/ui/Application.java

```diff
@@ -84,7 +84,7 @@ public void handleOpenFilesAction(Application app, long time, String files[]) {
         // currently used only on Mac OS X
         public void handleQuitAction(Application app, long time) {
         }
-        public void handlePreferencesChanged(Map<String, Object> preferences) {
+        public void handlePreferencesChanged(Map<String, Object> preferences, int suggestedDelayMillis) {
```

Comment:
I wonder if we should always debounce the changes with a short delay?
A delay of 100-150 ms will be acceptable from the user perspective.
What do you think?